src/testdir/test_diffmode.vim

```diff
@@ -1664,11 +1664,11 @@ func Test_diff_scroll_many_filler()
   call assert_equal(18, winsaveview().topfill)
   exe "normal! \<C-B>"
   redraw
-  call assert_equal(3, winsaveview().topline)
-  call assert_equal(19, winsaveview().topfill)
+  call assert_equal(1, winsaveview().topline)
+  call assert_equal(0, winsaveview().topfill)
   exe "normal! \<C-B>"
   redraw
-  call assert_equal(2, winsaveview().topline)
+  call assert_equal(1, winsaveview().topline)
   call assert_equal(0, winsaveview().topfill)
   exe "normal! \<C-B>"
   redraw
```

Comment:
This test may need more lines at the top, otherwise the last two `<C-B>` have no room to scroll.